Разметка всех страниц

afkmast3r opened the pull request:
Добавил тэги header, main, section.

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2052031/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2052031-sedona-34/2/

afkmast3r committed:
Верстка первой страницы

Добавил <header>, <main>, <section>.

user-ecom-git commented on index.html:
```diff
@@ -5,6 +5,125 @@
+              <img class="advantages-photo" src="#" alt="Дом с вывеской">
+            </div>
+          </li>
+          <!-- <li class="advantages-item">                               --- Первоначальный вариант
```
При отправке мр, нужно убирать комменты

user-ecom-git commented on index.html:
```diff
@@ -5,6 +5,125 @@
+          <li class="advantages-item">
+            <img class="advantages-icon" src="" alt="дом">
+            <h3 class="advantages-title">Жильё</h3>
+            <p>Рекомендуем пожитьв настоящем мотеле, всё как в кино!</p>
```
пожитьв -> пожить в

user-ecom-git commented on index.html:
```diff
@@ -5,6 +5,125 @@
+          <li class="advantages-item">
+            <div class="advantages-item-text-img">
+              <h3 class="advantages-title">Настоящий городок</h3>
+            <span>— №1 —</span>
```
В ситуациях, когда должен быть текст на одной строке, нужно пользоваться неразрывным пробелом. В данном случаее, необходимо по обе стороны номера поставить этот символ - &nbsp;

Вот тут можно об этом почитать - https://webstyle.sfu-kras.ru/nerazryvnyj-probel#:~:text=%D0%A7%D1%82%D0%BE%D0%B1%D1%8B%20%D0%B8%D0%B7%D0%B1%D0%B5%D0%B6%D0%B0%D1%82%D1%8C%20%D1%80%D0%B0%D0%B7%D1%80%D1%8B%D0%B2%D0%B0%20%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%B8%20%D0%BC%D0%B5%D0%B6%D0%B4%D1%83,%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D1%83%D1%8F%20%D0%BA%D0%BB%D0%B0%D0%B2%D0%B8%D0%B0%D1%82%D1%83%D1%80%D0%BD%D1%83%D1%8E%20%D0%BA%D0%BE%D0%BC%D0%B1%D0%B8%D0%BD%D0%B0%D1%86%D0%B8%D1%8E%20Alt%20%2B%200160%20.

user-ecom-git replied:
И также во всех подобных случаях

afkmast3r committed:
Доделывает index и создает catalog

Доделал футер, создал страницу каталог, скопировал код (хэадер, подписку и футер) в catalog.html

afkmast3r committed:
Добавляет изменения в catalog.html

Добавил хэадер, чекбоксы, радиокнопки.

afkmast3r committed:
Добавляет фильтр, карточки гостиниц, пагатинацию

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,273 @@
+        <h1 class="catalog-header-title">Гостиницы Седоны</h1>
+        <ul class="breadcrumbs">
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link" href="index.html">Главная</a>
```
А тут разве не изображение домика должно быть?

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,273 @@
+        <ul class="product-list">
+          <li class="product-card">
+            <a class="product-card-link" href="#">
+              <img class="product-card-image" src="" alt="">
```
Альт не оставляй пустым

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,273 @@
+        </ul>
+
+<!-- Пагатинация -->
+<ul class="pagination">
```
У тебя верстка поехала. Прогони валидатором, чтобы всё проверить

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,273 @@
+          </li>
+          <li class="product-card">
+            <a class="product-card-link" href="#">
+              <img class="product-card-image" src="" alt="">
```
То же самое насчет альта

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,273 @@
+          </li>
+          <li class="product-card">
+            <a class="product-card-link" href="#">
+              <img class="product-card-image" src="" alt="">
```
И тут

boumrz commented on catalog.html:
```diff
@@ -0,0 +1,273 @@
+          </li>
+          <li class="product-card">
+            <a class="product-card-link" href="#">
+              <img class="product-card-image" src="" alt="">
```
И тут

afkmast3r committed:
Исправляет ошибку тэга FORM, и по мелочам

Исправил ошибку в index.html и catalog.html а части незакрытого тэга FORM.  Отформатировал обе страницы в единый стиль. Добавил описание к изображениям.

afkmast3r committed:
Заменяет таблицу на спан в контейнере

afkmast3r committed:
Добавляет картинки на главную страницу и каталог
